trestle/resources/config.ini

```diff
@@ -0,0 +1,33 @@
+[key-model-mapping]
+groups = trestle.oscal.catalog.Group
```

Comment:
I don't understand what this is and why we would be codifying the mapping here in a config file. This should be in source as it's not user configurable.

i'm also confused as to why you would need anything beyond the top level models.

Reply:
This would be as everything else should be inferrable based on which of the top level models you are acting on.

Reply:
Remove key-model-mapping from config.ini. It is indeed inferrable.